compute/cloud-client/src/main/java/compute/CreateInstanceBulkInsert.java

```diff
@@ -0,0 +1,124 @@
+import com.google.cloud.compute.v1.InstancesClient;
+import com.google.cloud.compute.v1.ListInstancesRequest;
+import com.google.common.collect.Lists;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.StringJoiner;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class CreateInstanceBulkInsert {
+  public static void main(String[] args)
+          throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "your-project-id";
+    // Name of the zone to create the instance in. For example: "us-west3-b"
+    String zone = "zone-name";
+    // An Instance Template to be used for creation of the new VMs.
+    String templateName = "instance-template";
+    // The maximum number of instances to create.
+    int count = 3;
+    // The string pattern used for the names of the VMs. The pattern
+    // must contain one continuous sequence of placeholder hash characters (#)
+    // with each character corresponding to one digit of the generated instance
+    // name. Example: a name_pattern of inst-#### generates instance names such
+    // as inst-0001 and inst-0002. If existing instances in the same project and
+    // zone have names that match the name pattern then the generated instance
+    // numbers start after the biggest existing number. For example, if there
+    // exists an instance with name inst-0050, then instance names generated
+    // using the pattern inst-#### begin with inst-0051. The name pattern
+    // placeholder #...# can contain up to 18 characters.
+    String namePattern = "instance-name-pattern";
+    // (optional): The minimum number of instances to create. If no min_count is
+    // specified then count is used as the default value. If min_count instances
+    // cannot be created, then no instances will be created and instances already
+    // created will be deleted.
+    int minCount = 2;
+    // (optional): A dictionary with labels to be added to the new VMs.
+    Map<String, String> labels = new HashMap<>();
+
+    bulkInsertInstance(project, zone, templateName, count, namePattern, minCount, labels);
+  }
+
+  public static List<Instance> bulkInsertInstance(String project, String zone, String templateName,
+                                                  int count, String namePattern, int minCount,
+                                                  Map<String, String> labels)
+          throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    String sourceInstanceTemplate;
+    try (InstanceTemplatesClient client = InstanceTemplatesClient.create()) {
+      sourceInstanceTemplate = client.get(project, templateName).getSelfLink();
+    }
```

Comment:
Extend this try catch to the entire sample. 
And move `InstancesClient` initialization in line 95 to the try block in line 77.